server/home/homeDB.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
       // let cacheContains = redisClient.get(movieId)
       // if (cacheContains) {
       return new Promise((resolve, reject) => {
-        Movie.find({id: movieId})
+        Movie.find({currentId: movieId})
```

Comment:
I believe this should be "id" accoridng to the movie schema?

Reply:
it needs to be currentId because currentID is the property in the schema.  ID wont work.